src/services/Notifications/SaveTimePreference.ts

```diff
@@ -0,0 +1,15 @@
+import { useMutation } from 'react-fetching-library';
+
+import { Operations, Schemas } from '../../generated/OpenapiNotifications';
+
+type SaveTimePrefActionsParams = {
+    body: Schemas.LocalTime;
+}
+
+const updateTimePrefActionCreator = (params: SaveTimePrefActionsParams) => {
+    return Operations.OrgConfigResourceSaveDailyDigestTimePreference.actionCreator({
+        body: params.body
```

Comment:
I have a question here, wouldn't I need a bundleId param to be able to save it to the specific bundle? Because right now, when I input a time it is saving for all bundles...

Reply:
It's global to the account, not specific to a particular bundle so i think it's fine.